src/scripts/global-ray-loader.php

```diff
@@ -1,46 +1,63 @@
 <?php
 
-use Spatie\GlobalRay\Support\Dump;
-use Spatie\GlobalRay\Support\Ray;
+namespace Spatie\GlobalRay;
 
-try {
-    include_once __DIR__ . "/../Support/Ray.php";
-    $rayPharPath = Ray::getPharPath();
-    globalRayPharLoader($rayPharPath, [
-        'spatie/ray',
-        'spatie/yii-ray',
-        'spatie/craft-ray',
-        'spatie/laravel-ray',
-        'spatie/wordpress-ray',
-    ]);
+include_once __DIR__ . "/../Support/Ray.php";
+include_once __DIR__ . "/../Support/Dump.php";
 
-    include_once __DIR__ . "/../Support/Dump.php";
-    $dumpPharPath = Dump::getPharPath();
-    globalRayPharLoader($dumpPharPath, [
-        'laravel/framework',
-        'illuminate/support',
-        'symfony/var-dumper',
-    ]);
-} catch (Throwable $exception) {
-}
+use Spatie\GlobalRay\Support\Dump;
+use Spatie\GlobalRay\Support\Ray;
 
-function globalRayPharLoader(string $pharPath, array $unlessDetectedPackages)
+class PharLoader
```

Comment:
I created a class and gave this file a namespace so our global functions wouldn't have any collision with a developers global functions in their project. For example, `isRunningTinkerwell()`, could be a defined global function. Let me know if you'd like this adjusted 👍